sklearn/ensemble/tests/test_common.py

```diff
@@ -1,19 +1,28 @@
 import pytest
+import numpy as np
 
 from sklearn.base import clone
 from sklearn.base import ClassifierMixin
 from sklearn.base import is_classifier
 
 from sklearn.datasets import make_classification
 from sklearn.datasets import make_regression
-
+from sklearn.datasets import load_iris, load_diabetes
+from sklearn.impute import SimpleImputer
 from sklearn.linear_model import LogisticRegression, LinearRegression
 from sklearn.svm import LinearSVC, LinearSVR, SVC, SVR
+from sklearn.pipeline import make_pipeline
 from sklearn.ensemble import RandomForestClassifier, RandomForestRegressor
 
 from sklearn.ensemble import StackingClassifier, StackingRegressor
 from sklearn.ensemble import VotingClassifier, VotingRegressor
 
+iris = load_iris()
+X, y = iris.data[:, 1:3], iris.target
```

Comment:
What is the reason to get only a couple of features?

Reply:
No actual rational to it. I just picked up that example from [here](https://github.com/scikit-learn/scikit-learn/blob/0fb307bf39bbdacd6ed713c00724f8f871d60370/sklearn/ensemble/tests/test_voting.py#L34-L36).

Let me make it as all three features itself.